src/support/msw/shoppingListItems.ts

```diff
@@ -58,6 +58,81 @@ export const postShoppingListItemsServerError = rest.post(
   }
 )
 
+/**
+ *
+ * PATCH /shopping_list_items/:id
+ *
+ */
+
+export const incrementShoppingListItemSuccess = rest.patch(
+  `${BASE_URI}/shopping_list_items/:id`,
+  async (req, res, ctx) => {
+    const itemId: number = Number(req.params.id)
+    const item = allShoppingListItems.find(({ id }) => id === itemId)
+    const list = allShoppingLists.find(({ id }) => id === item?.list_id)
+
+    if (!item || !list) return res(ctx.status(404))
+
+    const allItems = shoppingListsForGame(list.game_id).flatMap(
+      ({ list_items }) =>
+        list_items.filter(({ description }) => description === item.description)
+    )
+
+    const aggListItem = { ...allItems[0], quantity: allItems[0].quantity + 1 }
+    const regItem = { ...item, quantity: item.quantity + 1 }
+
+    return res(ctx.status(200), ctx.json([aggListItem, regItem]))
+  }
+)
+
+export const decrementShoppingListItemSuccess = rest.patch(
+  `${BASE_URI}/shopping_list_items/:id`,
+  async (req, res, ctx) => {
+    const itemId: number = Number(req.params.id)
+    const item = allShoppingListItems.find(({ id }) => id === itemId)
+    const list = allShoppingLists.find(({ id }) => id === item?.list_id)
+
+    if (!item || !list) return res(ctx.status(404))
+
+    const allItems = shoppingListsForGame(list.game_id).flatMap(
+      ({ list_items }) =>
+        list_items.filter(({ description }) => description === item.description)
+    )
+    const aggListItem = { ...allItems[0], quantity: allItems[0].quantity - 1 }
+    const regItem = { ...item, quantity: item.quantity - 1 }
+
+    return res(ctx.status(200), ctx.json([aggListItem, regItem]))
+  }
+)
+
+// Returns the same validation errors regardless of request body
+export const updateShoppingListItemUnprocessable = rest.patch(
```

Comment:
This didn't end up being used in testing this feature but will be needed for the next one so I'm leaving it.